### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by CapriciousRebel**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,11 +2,17 @@
 FROM python:3.9
 
 # Set working directory.
+WORKDIR /src
 
 # Expose port 8000 for your FastAPI server.
+EXPOSE 8000:8000
```
Comment (shubhankar-ti) [inline Dockerfile]: Why is the `:8000` part needed here?

Comment (Smith-TI) [reply Dockerfile]: Must have been a typo, fixing it.

Comment (Smith-TI) [reply Dockerfile]: Fixed and pushed

**Comment by Smith-TI** [comment]
**Setup instructions:**

- Build the docker image: `docker build -t assignment .`
- Run the docker image: `docker run -p 8000:8000 -t -i assignment`
- Go to http://localhost:8000/

**Review by shubhankar-ti — APPROVED** [review]

**Pull request merged**